projects/adxrs290-pmdz/src.mk

```diff
@@ -41,4 +42,5 @@ SRCS += $(DRIVERS)/api/no_os_gpio.c     \
         $(NO-OS)/util/no_os_list.c      \
         $(NO-OS)/util/no_os_util.c      \
         $(NO-OS)/util/no_os_alloc.c     \
+        $(DRIVERS)/api/no_os_pwm.c     \
```

Comment:
is this needed?

Reply:
The adxrs290-pmdz project has the TIM module enabled in the .ioc file. Hence added the PWM drivers here as we are making a check for the HAL_TIM_MODULE_ENABLED in the stm32_spi.c driver